container_test.go

```diff
@@ -1783,6 +1864,10 @@ func TestContainerExecLargeOutputWithTTY(t *testing.T) {
 func TestShortRunningTaskPid(t *testing.T) {
 	t.Parallel()
 
+	if runtime.GOOS == "darwin" {
+		t.Skip("test with true command isn't supported on darwin")
+	}
+
```

Comment:
Should we just exclude this file on darwin? Seems to be a lot cleaner solution.

Reply:
thanks for the comment.

As I commented at https://github.com/containerd/containerd/pull/4526#discussion_r494017023, there are still valid tests in `container_test.go` even with darwin and it's worthwhile to have those tests I believe.

Excluding the tests is indeed much cleaner but I'd like to test if it's fine.